.github/workflows-source/schedule-daily.yml

```diff
@@ -135,7 +135,7 @@ jobs:
   fi-tests-nightly:
     name: Bazel Test FI Nightly
     <<: *dind-large-setup
-    timeout-minutes: 120
+    timeout-minutes: 240
```

Comment:
With these changes, the FI nightly tests timed out after 2h, so bumping this to 4h. Will consider options before merging if the nightly test passes with this setting.

Reply:
[This more recent run](https://github.com/dfinity/ic/actions/runs/10610013902) completed in 73min, so it looks like it is quite dependent on the busyness of the hardware it is running on? I'd still leave in the longer timeout for now.